include/config.h

```diff
@@ -227,11 +227,12 @@ struct sway_config {
 		uint32_t background;
 	} border_colors;
 
-	// floating view minimum
+	// floating view
 	int32_t floating_maximum_width;
 	int32_t floating_maximum_height;
 	int32_t floating_minimum_width;
 	int32_t floating_minimum_height;
+	char *floating_minimum_size;
```

Comment:
Initially I was using a string as the default; something like ("75 x 50"); but I changed my mind and forgot to do the clean up.